docs/source/tutorial.rst

```diff
@@ -112,15 +122,18 @@ First, use git to create a new branch under this naming scheme:
 For instance, the author's branch would be named
 ``kevin-fu/robocup-sw-tutorial``.
 
-Then take a look at the defense play in
-``rj_gameplay/rj_gameplay/play/defense.py``. Launch soccer (our UI) and the
-ER-force simulator, same way as you did in the installation guide, then select
-this play as the test play to see it in action. Click the green checkmark in our
-UI. You should see 3 robots form a wall, 2 robots mark the opposing team, and 1
-robot play goalie.
+Launch soccer (our UI) and the ER-force simulator, same way as you did in the 
+installation guide. Press the green check mark. You should see four wallers
+and one goalie move into position. Click anywhere on the field to place the 
+ball in that location. You should see all five robots move between the ball
+and the goal.
 
-Figure out which line(s) to change so that 4 robots form a wall instead of 3.
-When done, take a screenshot of the four wallers.
+Open the file ``soccer/src/soccer/strategy/agent/position/waller.cpp``. 
+Find the line of code that calculates the ``wall_spacing`` and double its value.
+
+Re-build the project (``make again``) and run the simulator again. You should
+see the wallers more spread out. Note that this is probably a less effective wall!
+This change is just for educational purposes. Take a screenshot of your new wall.
```

Comment:
```suggestion
This change is just for educational purposes. 

**Take a screenshot of your new wall.***
```

Reply:
I think that we should make the action parts of the tutorial bolded so that they aren't missed.